### PR TITLE
Add a Multi-Cluster example of changing the placement of an application

### PR DESCRIPTION
**Pull request by mgianatagh**
# Description

An en example to demonstrate how to change the placement of an application

Fixes VZ-2232

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: examples/multicluster/change-placement/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Change placement of an application
+
+This example shows how to change the placement of an application from one cluster to another.
+
+## Prerequisites
+
+The multicluster Hello World Helidon example will used to show how to change the placement of an application to another cluster.  Follow the [instructions](../hello-helidon/README.md) to deploy the multicluster Hello World Helidon example.
```
Comment (rosemarymarano) [inline examples/multicluster/change-placement/README.md:7]: will used -> will be used
Follow -> First, follow

File: examples/multicluster/change-placement/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    ```
+    $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl apply -f mc-hello-helidon-comp.yaml
+    $ KUBECONFIG=$KUBECONFIG_ADMIN kubectl apply -f mc-hello-helidon-app.yaml
+    ```
+
+## Verify the change in placement
+
+Wait a few minutes for the change in placement to take effect on each cluster.
+
+1. Use the following commands to verify the resources have been removed from the managed cluster.
```
Comment (rosemarymarano) [inline examples/multicluster/change-placement/README.md:25]: to verify the -> to verify that the

File: examples/multicluster/change-placement/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    ```
+    $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl get MultiClusterComponent hello-helidon-component -n hello-helidon
+    $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl get mccomp -n hello-helidon
+    $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl get MultiClusterApplicationConfiguration hello-helidon-appconf -n hello-helidon
+    $ KUBECONFIG=$KUBECONFIG_MANAGED1 kubectl get VerrazzanoHelidonWorkload -n hello-helidon
+    ```
+
+1. Use the following commands to verify the resources have been moved to the admin cluster.
```
Comment (rosemarymarano) [inline examples/multicluster/change-placement/README.md:34]: to verify the -> to verify that the

**Review by rosemarymarano — DISMISSED** [review]
Minor edits.

File: examples/multicluster/change-placement/README.md (hunk 1 above)
Comment (mgianatagh) [reply examples/multicluster/change-placement/README.md:7]: fixed in next push

File: examples/multicluster/change-placement/README.md (hunk 2 above)
Comment (mgianatagh) [reply examples/multicluster/change-placement/README.md:25]: fixed in next push

File: examples/multicluster/change-placement/README.md (hunk 3 above)
Comment (mgianatagh) [reply examples/multicluster/change-placement/README.md:34]: fixed in next push

File: examples/multicluster/change-placement/mc-hello-helidon-app.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                metadata:
+                  name: hello-helidon-ingress
+                spec:
+                  rules:
+                    - paths:
+                        - path: "/greet"
+                          pathType: Prefix
+  placement:
+    clusters:
+      - name: local
```
Comment (michael-cico) [inline examples/multicluster/change-placement/mc-hello-helidon-app.yaml:35]: Should this be "managed1" to start?  The doc above says to change this from "managed1" to "local" to do the move.

Comment (michael-cico) [reply examples/multicluster/change-placement/mc-hello-helidon-app.yaml:35]: OK, never mind, the instructions say to run the HH example first, then apply these I see.

**Review by michael-cico — APPROVED** [review]

**Pull request merged**